hack/update-codegen.sh

```diff
@@ -24,6 +24,7 @@ IMAGE_NAME="antrea/codegen:kubernetes-1.21.0-build.1"
 function docker_run() {
   docker pull ${IMAGE_NAME}
   docker run --rm \
+  -e GOPROXY=https://goproxy.cn \
```

Comment:
use a more generic way to make the script take an optional proxy, instead of hardcoding one specific to an area

Reply:
sure, I forgot to remove this line. let me add proxy support in another PR.